lux/executor/PandasExecutor.py

```diff
@@ -394,6 +394,18 @@ def execute_2D_binning(vis: Vis) -> None:
             x_attr = vis.get_attr_by_channel("x")[0].attribute
             y_attr = vis.get_attr_by_channel("y")[0].attribute
 
+            if vis.data[x_attr].dtype == np.dtype('O'):
```

Comment:
This is not a great detection mechanism for mixed type. There's a lot of cases where the column is properly detected as object (e.g. string). See [here](https://github.com/lux-org/lux/blob/master/lux/core/series.py#L151) for a better detection mechanism.

Reply:
This is probably okay for now since this code only goes through 2D binning which is generally all quantitative values to begin with.

Reply:
Turns out the object type check is a more optimized approach than [pandas.api.types.is_object_dtype](https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_object_dtype.html), so we're all good on this front!
![Screen Shot 2022-02-07 at 7 13 29 AM](https://user-images.githubusercontent.com/5554675/152815485-6521d812-e0d6-40c7-b5e4-105bce9a0adc.png)